<ui version="4.0" >
 <class>FileWinBase</class>
 <widget class="QDialog" name="FileWinBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Load files</string>
  </property>
  <widget class="QLabel" name="FNLabel" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>60</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font/>
   </property>
   <property name="text" >
    <string>File name:</string>
   </property>
   <property name="wordWrap" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="ModuleLabel" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>0</y>
     <width>440</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font" >
    <font/>
   </property>
   <property name="text" >
    <string>load files for unknown module</string>
   </property>
   <property name="alignment" >
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="FileType" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>30</y>
     <width>460</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="BrwsButt" >
   <property name="geometry" >
    <rect>
     <x>400</x>
     <y>90</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text" >
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="FileName" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>90</y>
     <width>381</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text" >
    <string>No file has been selected</string>
   </property>
  </widget>
  <widget class="QLabel" name="ComLabel" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>130</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>Label</string>
   </property>
   <property name="wordWrap" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="OKButt" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>200</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font" >
    <font/>
   </property>
   <property name="text" >
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Comments" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>160</y>
     <width>460</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font" >
    <font/>
   </property>
   <property name="toolTip" >
    <string>Label that will show up in tree-view after data uploading</string>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="alignment" >
    <set>Qt::AlignLeading</set>
   </property>
  </widget>
  <widget class="QPushButton" name="CancButt" >
   <property name="geometry" >
    <rect>
     <x>400</x>
     <y>190</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font" >
    <font/>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
